### ISSUE TITLE
Titles for graphs

### ISSUE THREAD

**Issue by saaaallYao** [WilliamYLu/EEP153Proj1#2]
Maybe adding a title or providing more background information for each graph can help to understand the graph better.

**Comment by Jaydenm5** [comment]
I agree, the graphs that are titled and labeled accordingly come off as a lot more "put together" than the ones without.
